kickstart/provision.erb

```diff
@@ -143,6 +144,11 @@ echo "updating system time"
 # update all the base packages from the updates repository
 yum -t -y -e 0 update
 
+<% if chef_enabled %>
+echo "Bootstraping chef"
+<%= respond_to?(:chef_bootstrap) ? chef_bootstrap(@host) : snippet_if_exists("chef-client omnibus bootstrap") %>
```

Comment:
I'd prefer not to because
* foreman_chef is not released on regular basis and may need to change template from release to release I need better control over these templates, this may change later
* I believe this belongs to plugin itself, these templates would be useless without plugin (which is difference to salt IIUC, salt uses global parameters, these templates rely on bootstrap helper provided by the plugin)